src/Files.Uwp/ViewModels/SettingsViewModels/AppearanceViewModel.cs

```diff
@@ -110,11 +111,8 @@ public AppTheme SelectedTheme
         /// <summary>
         /// Forces the application to use the correct resource styles
         /// </summary>
-        private async void UpdateTheme()
+        private void UpdateTheme()
         {
-            // Allow time to remove the old theme
-            await Task.Delay(250);
```

Comment:
Have you confirmed that the delay is no longer necessary?

Reply:
I understand it was needed when changing custom theme, due to the call to App.ExternalResourcesHelper.UpdateTheme() not being awaited (now it is) but I did not write that code so I may have misunderstood its purpose.